databasebuild.yaml

```diff
@@ -0,0 +1,19 @@
+#To use this you will need buildcomponents.yaml in your top level ci file. 
+
+build:
+  extends: .migrate
+  stage: build
+  services:
+    - $POSTGRES_VERSION
+  after_script:
+    - npm test
```

Comment:
Is this job intended to be used for build-testing?

Reply:
this should run the test but we've since learned this should change. 

https://gitlab.com/gitlab-org/gitlab-ce/issues/43010